pkg/types/current/types.go

```diff
@@ -22,27 +22,26 @@ import (
 	"os"
 
 	"github.com/containernetworking/cni/pkg/types"
-	types020 "github.com/containernetworking/cni/pkg/types/020"
+	types040 "github.com/containernetworking/cni/pkg/types/040"
 	"github.com/containernetworking/cni/pkg/types/convert"
 )
 
-const ImplementedSpecVersion string = "0.4.0"
+const ImplementedSpecVersion string = "1.0.0"
 
-var SupportedVersions = []string{"0.3.0", "0.3.1", ImplementedSpecVersion}
+var SupportedVersions = []string{ImplementedSpecVersion}
```

Comment:
Is this right?  Looks like we would still support 0.3.x and 0.4.0.

Reply:
No, because 1.0.0 doesn't have the "Version" field in the IPConfig struct and perhaps we'll make other changes before final.

Reply:
Follow-up: Bryan was really asking why the plugins export this at all; there are only two uses in the reference plugins and those are for error-checking and can be trivially replaced.